queries.sql

```diff
@@ -8,3 +8,59 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- Start a transaction
+BEGIN;
+
+-- Delete all animals born after Jan 1st, 2022
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+-- Create a savepoint for the transaction
+SAVEPOINT SP1;
+
+-- Update all animals' weight to be their weight multiplied by -1
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+
+-- Rollback to the savepoint
+ROLLBACK TO SP1;
+
+-- Update all animals' weights that are negative to be their weight multiplied by -1
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+
+-- Commit transaction
+COMMIT;
```

Comment:
- [ ] Please add these queries
- `UPDATE` the animal's table by setting the species column to unspecified, then roll back the change and remember to `verify that the change was made` and `verify that change was undone`

Reply:
- [ ] `UPDATE` the animal's table by setting the species column to digimon for all animals that have a name ending in mon.
- [ ] `UPDATE` the animals table by setting the species column to Pokemon for all animals that don't have species already set.
- [ ] Commit the transaction.